.github/workflows/build.yml

```diff
@@ -198,6 +211,20 @@ jobs:
         run: cmake --build build --target tests
       - name: Package
         run: cmake --build build --target package
+      - name: Unpack
+        run: |
+          mkdir build/install
+          7z x -obuild/install build/lmms-*.exe && find build/
```

Comment:
The usage of `find` looks redundant. Since these commands will run on `bash`, I guess `test -d build/` will work here.

Reply:
I doubt that this `find` call was used to test if build was created, because it already exists. I guess this was added to simply list all the installed files, which `test -d` does not. I really do not remember who suggested this code line to me, maybe it was @tresf ?

Anyways, the only sense of the `find` is to list the files. In my opinion, this is not necessary, I will remove this if no one disagrees.

Reply:
... but won't `test -d`  falsely succeed if it's empty?

Reply:
... IIRC, the justification for this was to find out if the installer was empty or not.

Reply:
... and I'd normally NOT have mixed `bash` with `pwsh`/`cmd`

Reply:
Well, indeed, `test -d` will succeed if it's empty. But the previous `mkdir build/install` will guarantee that `build/install` exists. So the find does not increase the stability - it simply lists all the files.

Reply:
Good points. I'm now okay with this.

Reply:
> Well, indeed, `test -d` will succeed if it's empty. But the previous `mkdir build/install` will guarantee that `build/install` exists. So the find does not increase the stability - it simply lists all the files.

Hmm... So the idea of testing gets tossed as a result of a faulty test?

Reply:
Sorry, tres, I do not understand what you mean.

Reply:
> So we have decided to remove the ability to test for an empty installer because the test for an empty installer was incorrect?

That is correct, that code line will not detect anything, because it will always succeed. Also, `7z` will return with an error code anyways in case the extraction failed, so this would abort the GitHub action script anyways.

Reply:
> Also, `7z` will return with an error code anyways in case the extraction failed

For clarification, if the NSIS installer is created successfully, but is empty, `7z` won't return an error.  With regards to the likeliness of this happening, I'm unsure.  The NSIS script that's being run is through a [cmake/cpack integration](https://cmake.org/cmake/help/latest/cpack_gen/nsis.html) which produces a `.nsi` script with programmatically-generated steps of which are rarely seen/reviewed by a human.  The likelihood of an empty installer would depend on the likeliness of one more steps NOT failing the `makensis` process.

Reply:
We could, alternatively, test for `build/install/lmms.exe`. What about that?